src/Commands/Multidev/CreateCommand.php

```diff
@@ -17,6 +17,7 @@ class CreateCommand extends TerminusCommand implements SiteAwareInterface
      * @authorized
      *
      * @name multidev:create
+     * @alias env:create
```

Comment:
Not sure what your experience is but i needed to use the plural `@aliases` annotation for these to show up in help and be registered in order to work.